src/cache/precomputed_quantities.jl

```diff
@@ -354,13 +355,12 @@ function eddy_diffusivity_coefficient(
         K = K_b * (z / f_b / h) * (1 - (z - f_b * h) / (1 - f_b) / h)^2
         return K
     else
-        return FT(0)
+        return eddy_diffusivity_coefficient(C_E, uₐ, z, ᶜp)
```

Comment:
Do we need this change? If not I think it would be better not to have it, as this may case a discontinuity in diffusivity.